src/main/resources/edu/cornell/kfs/coa/cu-ojb-coa.xml

```diff
@@ -2070,4 +2070,114 @@
+			<foreignkey field-ref="financialFederalFundedCode" />
+		</reference-descriptor>
+		<reference-descriptor name="financialObjectLevel" class-ref="org.kuali.kfs.coa.businessobject.ObjectLevel"
+			auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true">
+			<foreignkey field-ref="chartOfAccountsCode" />
+			<foreignkey field-ref="financialObjectLevelCode" />
+		</reference-descriptor>
+		<reference-descriptor name="chartOfAccounts" class-ref="org.kuali.kfs.coa.businessobject.Chart" auto-retrieve="true"
+			auto-update="none" auto-delete="none" proxy="true">
+			<foreignkey field-ref="chartOfAccountsCode" />
+		</reference-descriptor>
+		<reference-descriptor name="reportsToChartOfAccounts" class-ref="org.kuali.kfs.coa.businessobject.Chart" auto-retrieve="true"
+			auto-update="none" auto-delete="none" proxy="true">
+			<foreignkey field-ref="reportsToChartOfAccountsCode" />
+		</reference-descriptor>
+		<reference-descriptor name="reportsToFinancialObject" class-ref="org.kuali.kfs.coa.businessobject.ObjectCode"
+			auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true">
+			<foreignkey field-ref="universityFiscalYear" />
+			<foreignkey field-ref="reportsToChartOfAccountsCode" />
+			<foreignkey field-ref="reportsToFinancialObjectCode" />
+		</reference-descriptor>
+		<reference-descriptor name="financialObjectType" class-ref="org.kuali.kfs.coa.businessobject.ObjectType"
+			auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true">
+			<foreignkey field-ref="financialObjectTypeCode" />
+		</reference-descriptor>
+		<reference-descriptor name="financialObjectSubType" class-ref="org.kuali.kfs.coa.businessobject.ObjectSubType" 
+			auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true">
+			<foreignkey field-ref="financialObjectSubTypeCode" />
+		</reference-descriptor>
+		<reference-descriptor name="universityFiscal" class-ref="org.kuali.kfs.sys.businessobject.SystemOptions"
+			auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true">
+			<foreignkey field-ref="universityFiscalYear" />
+		</reference-descriptor>
+
+		<reference-descriptor name="extension" class-ref="edu.cornell.kfs.coa.businessobject.ObjectCodeExtendedAttribute" auto-retrieve="true" 
+			auto-update="object" auto-delete="object" proxy="false">
+			<foreignkey field-ref="universityFiscalYear" />
+			<foreignkey field-ref="chartOfAccountsCode" />
+			<foreignkey field-ref="financialObjectCode" />
+		</reference-descriptor>
+
+		<collection-descriptor name="balances" element-class-ref="org.kuali.kfs.gl.businessobject.Balance"
+			collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" auto-retrieve="true" auto-update="object" auto-delete="false" proxy="false">
+			<inverse-foreignkey field-id-ref="1" field-ref="universityFiscalYear" />
+			<inverse-foreignkey field-id-ref="2" field-ref="chartOfAccountsCode" />
+			<inverse-foreignkey field-id-ref="5" field-ref="objectCode" />
```

Comment:
Is there a reason why both "field-id-ref" and "field-ref" are being specified? I thought that only one of them needed to be used.

Reply:
it's the only way I could get the join to work

Reply:
Ok, thanks for clarifying.